docs/design.md

```diff
@@ -31,11 +31,13 @@ and its initialization can also be delegated to the application code.
 ### Unsupported scenarios
 
 - **Applications using Ahead-of-Time (AOT) compilation**:
-The current implementation relies on the [CLR Profiler APIs](https://docs.microsoft.com/en-us/dotnet/framework/unmanaged-api/profiling/)
-and doesn't support AOT.
+The current implementation relies on the [host startup hook](https://github.com/dotnet/runtime/blob/main/docs/design/features/host-startup-hook.md)
+or the [CLR Profiler APIs](https://docs.microsoft.com/en-us/dotnet/framework/unmanaged-api/profiling)
+and neither is support when publishing AOT compiled applications.
```

Comment:
```suggestion
and neither is supported when publishing AOT compiled applications.
```

Reply:
Might be worth spelling out AOT in full the first time.

Reply:
@theletterf a few lines above AOT is spelled out, do you think is still desirable to spell it out again here?